src/events/mod.rs

```diff
@@ -116,6 +117,12 @@ impl EventHandler for Handler {
             reaction_add::reaction_add(ctx, event).await
         );
     }
+    async fn reaction_remove(&self, ctx: client::Context, event: Reaction) {
+        log_error!(
+            "Error while handling reaction_remove event",
+            reaction_remove::reaction_remove(ctx, event).await
+        );
+    }
```

Comment:
Won't this trigger on _every_ removal, not only when the last reaction get's removed? in that case this would reduce the emoji count further than it should.

also, is this even necessary? As in, I doubt that the amount of reactions that get added and then removed again is in any way statistically significant in this context, and it does add a considerable amount of code,.... Did anyone explicitly request this addition? Not sure if it's worth the code otherwise ^^'

Reply:
> also, is this even necessary? As in, I doubt that the amount of reactions that get added and then removed again is in any way statistically significant in this context

:gremlin:

and yes reaction add-remove spam is rather common.

Reply:
bruh people are stupid,... but fair i guess.